server/cluster/prepare_checker.go

```diff
@@ -40,7 +40,7 @@ func newPrepareChecker() *prepareChecker {
 func (checker *prepareChecker) check(c *core.BasicCluster) bool {
 	checker.RLock()
 	defer checker.RUnlock()
-	if checker.isPrepared || time.Since(checker.start) > collectTimeout {
+	if checker.prepared || time.Since(checker.start) > collectTimeout {
```

Comment:
Is it possible that the condition `time.Since(checker.start) > collectTimeout` comes true before `prepared`?  What will happen if `checker.prepared` never comes true?